packages/relay/src/lib/eth.ts

```diff
@@ -1288,7 +1290,7 @@ export class EthImpl implements Eth {
       } else {
           params.timestamp = [];
 
-          const fromBlockResponse = await this.getHistoricalBlockResponse(fromBlock || "latest", true, requestId);
+          const fromBlockResponse = await this.getHistoricalBlockResponse(toBlock && !fromBlock ? toBlock : fromBlock || "latest", true, requestId);
```

Comment:
Pull the first param calculation out into a variable for improved readability.

Reply:
Good idea. Done.